dds_web/api/user.py

```diff
@@ -232,6 +195,76 @@ def add_user_to_project(existing_user, project, role):
             ),
         }
 
+    @staticmethod
+    @logging_bind_request
+    def compose_and_send_email_to_user(userobj, mail_type, link=None, project=None):
+
+        # Compose and send email
+        unit_name = None
+        project_id = None
+        deadline = None
+        if auth.current_user().role in ["Unit Admin", "Unit Personnel"]:
+            unit = auth.current_user().unit
+            unit_name = unit.external_display_name
+            unit_email = unit.contact_email
+            sender_name = auth.current_user().name
+            subject_subject = unit_name
+
+        else:
+            sender_name = auth.current_user().name
+            subject_subject = sender_name
+
+        # Fill in email subject with sentence subject
+        if mail_type == "invite":
+            subject = f"{subject_subject} invites you to the SciLifeLab Data Delivery System"
+            recepients = [userobj.email]
+        elif mail_type == "project_release":
+            subject = f"Project made available by {subject_subject} in the SciLifeLab Data Delivery System"
+            recepients = [x.email for x in userobj.emails]
+            project_id = project.public_id
+            deadline = project.current_deadline.astimezone(datetime.timezone.utc).strftime(
+                "%Y-%m-%d %H:%M:%S %Z"
+            )
```

Comment:
Either add an `else` or have `if mail_type not in ["invite", "project_release"]: Raise <something>`

Reply:
Since the arguments for this function will only come from within the code, do we really need it?

Reply:
I think we should add it just in case, since this function can be called from outside, and it's just better to not leave anything to chance. 👍🏻

Reply:
Done!